stylesheets/_structure.scaffolding.scss

```diff
@@ -30,7 +25,8 @@ body {
   clear: both;
   display: table;
   height: 100%;
-  margin: 0;
+  // Allow any content to clear the fixed footer
+  margin: 0 0 40px;
```

Comment:
Moved the padding to be a margin on the `.container` class so it's not added to the sign in page.